wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - nominations
+  - nom
+  - noms
+  - heart
+---
+
+# Nomination
+
+*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*
+
+::: Infobox
+![](img/nomination-progress-beatmapset1857663.png "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
+:::
+
+A **nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
```

Comment:
```suggestion
A **nomination** is a formal vote of approval coming from [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [Ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [Ranking Criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
```

Reply:
hesitant to capitalize ranked as it's a regular adjective here, not the category name -- you don't say that an article is Written, or that a game is Played

Reply:
applied the rest